### DISCUSSION TITLE
Is the rightclick invalid？

### DISCUSSION THREAD

**Discussion by lykl** [pixijs/pixijs#9140] [Q&A]
The rightclick  only fires the Click event.

**Comment by SuperSodaSea** [comment]
Yeah this seems to be a bug (example in v7.1.2: <https://pixiplayground.com/#/edit/toLp5QV_d2FWPoyA5lORs>).
Would you like to create a [bug report issue](https://github.com/pixijs/pixijs/issues/new?assignees=&labels=&template=bug_report.yml&title=Bug%3A+) for this?
